piksi_tools/console/settings.yaml

```diff
@@ -1391,6 +1391,28 @@
   Description: Enable NTRIP client. The interface will receive 1002, 1004, 1005, 1006, 1010, 1012, 1033 and 1230 RTCMv3.1 messages and will not transmit or receive any other messages.
   Notes: If True, NTRIP client will be used.
 
+- group: ntrip
+  name: username
+  expert: false
+  type: string
+  units: N/A
+  default value: N/A
+  readonly: false
+  enumerated possible values:
+  Description: NTRIP username to use. The interface will receive 1002, 1004, 1005, 1006, 1010, 1012, 1033 and 1230 RTCMv3.1 messages and will not transmit or receive any other messages.
+  Notes: Username to use with NTRIP client. NTRIP must be enabled to use this setting.
```

Comment:
Can you give some guidance as to what special characters are / are not allowed here?